streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamFlatMapTest.java

```diff
@@ -74,7 +71,7 @@ public void testFlatMap() {
 
         assertEquals(6, supplier.theCapturedProcessor().processed.size());
 
-        final String[] expected = {"10:V1", "20:V2", "21:V2", "30:V3", "31:V3", "32:V3"};
+        final String[] expected = {"10:V1 (ts: 0)", "20:V2 (ts: 0)", "21:V2 (ts: 0)", "30:V3 (ts: 0)", "31:V3 (ts: 0)", "32:V3 (ts: 0)"};
```

Comment:
The test of the expected result, also checks the timestamp now. Similar in other tests.